backend/User.java

```diff
@@ -0,0 +1,226 @@
+package backend;
```

Comment:
No such thing as a "frontend" in the context of running our Java code. Consequently, I don't think there's a need for an explicit "backend" package to exist.

Reply:
If no such package is necessary, please remove from final commit before approved PR